documentation/src/docs/metrics.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Pravega Metrics
```

Comment:
Same problem with titles: "Example for Starting a Metric Service" (and other section titles) appear super large and is not in right index.

Reply:
fixed title sizes and right-side index